app/poc-common/form-display/directives/formField.js

```diff
@@ -35,20 +35,19 @@ angular.module('poc.common.formdisplay')
                     if(last.value.datatype && last.value.datatype.display === "Coded") {
                         $scope.fieldModel.value = realValueOfField($scope.fieldModel.fieldConcept.concept.answers, last.value);
                         return;
-                        console.log($scope.fieldModel.value);
+                       
                     }
                     $scope.fieldModel.value = last.value; 
                }
             });
         };
 
-        formLogic.getYesNo = function (param) {
-          
-           if(param==$scope.fieldUuid && !_.isEmpty(param)){
-           /// $scope.fieldModel.value=$scope.fieldModel.value[1]; 
-            //return;
-            
-           }
+        formLogic.seeData = function (data) {
+
+          if (valueYes == "e1d81b62-1d5f-11e0-b929-000c29ad1d07"){
+           
+
+          }
         };
```

Comment:
What about this function @Benilde? It is doing nothing, can we remove or you're still coding?